Samsung/TAU-Design-Editor
# Change DE icon for remote samples

[Issue] http://suprem.sec.samsung.net/jira/browse/TIZENWF-2660
[Problem] Default DE icon should not be present once sample
    is opened using /demos API.
[Solution] Show fa-play.png icon which was generated from http://fa2png.io/
    To use FontAwesome directyly in this extension more changes would be needed.

[TEST 1]
1. Open http://localhost:3000/demos/?path=mobile%2FUIComponents%2Fcomponents%2Fcontrols%2Fcheckbox.html
2. Live preview icon should shown (after clicking, it should be green).

[TEST 2]
1. Open http://localhost:3000/
2. Login and create sample web application.
3. Open the project.
4. Desktop icon should shown (after clicking, it should be green).
